homeassistant/components/denonavr/media_player.py

```diff
@@ -249,9 +249,29 @@ def __init__(
             and MediaPlayerEntityFeature.SELECT_SOUND_MODE
         )
 
+        self._receiver.register_callback("ALL", self._telnet_callback)
+
+    async def _telnet_callback(self, zone, event, parameter):
+        """Process a telnet command callback."""
+        if zone != self._receiver.zone:
+            return
+
+        self.async_schedule_update_ha_state(False)
```

Comment:
```suggestion
        self.async_write_ha_state()
```

Assuming the object already has the new state, we don't need to schedule an update and can just write the state.

Reply:
It does, good catch. Addressed in https://github.com/home-assistant/core/pull/85980/commits/5ffb5dd17e8236b36bcdb20169868be35895d3e6